mediathread/templates/courseaffils/course_update_form.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html' %}
+{% load bootstrap3 %}
+
+{% block title %}&mdash; Course Settings{% endblock %}
+
+{% block content %}
+    <h1>Course Settings: {{object.title}}</h1>
```

Comment:
This can be an h2 to match the rest of the dashboard pages. {{object.title}} can be removed.